assets/RFCs/README.md

```diff
@@ -0,0 +1,9 @@
+# Internet Engineering Task Force (IETF) Request for Comments (RFC)
+
+This folder is dedicated to storing relevant RFCs. If you would like to add an RFC to this folder, please submit a pull request adding it!
+
+According to the [IETF's FAQ](https://trustee.ietf.org/about/faq/#collapse3103):
+
+> Am I allowed to reproduce whole RFCs?
+> 
+> Yes. Since the beginning of the RFC series, reproduction of whole RFCs (including translation into a language other than English) has been allowed and encouraged. The IETF Trust and the RFC Editor place no restrictions on this. Most RFCs include the standard phrase “Distribution of this memo is unlimited” to indicate this.

```

Comment:
I think I would prefer to leave this file out, as I would rather not encourage people to add RFCs (it should be a special case, not a common case I think).

Reply:
Hmm, it appears the proposal here isn't to just add these two RFCs, but rather introduce a process by which any RFC can be imported.  I'm in favor of bringing in the two RFCs, I will need to think more on creating a path toward inclusion of *any* RFC.  I am not strongly against it, though I do worry a bit about "you allow RFCs, why not also support my-new-standard-repository-that-i-created-yesterday?"

Reply:
> I do worry a bit about "you allow RFCs, why not also support my-new-standard-repository-that-i-created-yesterday?"

I think this is more of a theoretical problem than one that is actually likely to occur.

Reply:
I wish that was the case, but I have had multiple people who are owners of other standards repositories ask to reference their standards repository (e.g., for some other blockchain) in their EIPs.  😖

Reply:
Well then, we can just... say no to them (or have them convert all of their standards repo into EIPs!)